plugins/WindowExplorer/prpsh.h

```diff
@@ -81,7 +81,7 @@ PPV_PROPPAGECONTEXT PvCreatePropPageContextEx(
     _In_opt_ PVOID Context
     );
 
-BOOLEAN PvPropPageDlgProcHeader(
+BOOLEAN _Success_(return) PvPropPageDlgProcHeader(
```

Comment:
Multiple statements need to be multi-line. For example:

FORCEINLINE
_Success_(return)
BOOLEAN